src/components/Scrollbar.tsx

```diff
@@ -20,15 +17,59 @@ export default function ({
   maxHeight = 350,
   fade = 'both',
 }: ChildrenProp & ScrollbarProps) {
-  const scrollRef = useRef() as MutableRef<HTMLDivElement>
-  const { isOnTop, isOnBottom } = useIsOverflow(scrollRef, maxHeight)
+  const wrapRef = useRef() as MutableRef<HTMLDivElement>
+  const thumbRef = useRef() as MutableRef<HTMLDivElement>
+  const [thumbHeight, setThumbHeight] = useState(100)
+
+  const { overflows, scrollMaxHeight, isOnTop, isOnBottom } = useIsOverflow(
+    wrapRef,
+    maxHeight
+  )
+
+  const handleScroll = () => {
+    if (!thumbRef.current || !wrapRef.current) return
+    const wrapCurrent = wrapRef.current
+
+    // .scrollTop counts whole box, not only visible
+    // we should divide whole box by the visible to get how many visible boxes are in scrollable container
+    const numberOfViews = wrapCurrent.scrollHeight / wrapCurrent.clientHeight
+    const scroll = wrapCurrent.scrollTop / numberOfViews
+
+    thumbRef.current.style.top = scroll + 'px'
+  }
+
+  useEffect(() => {
+    const numberOfViews =
+      wrapRef.current.scrollHeight / wrapRef.current.clientHeight
+
+    setThumbHeight(100 / numberOfViews)
```

Comment:
This is not responsive, if we'll add some child nodes it won't update the `scrollThumb` height, also seems to be buggy with card, that is more than wrapper